components/spider-proto/storage/storage.proto

```
@@ -0,0 +1,71 @@
+syntax = "proto3";
+
+package storage;
```

Buf (1.70.0)

[error] Files with package "storage" must be within a directory "storage" relative to root but were in directory "components/spider-proto/storage".

(PACKAGE_DIRECTORY_MATCH)